benchmark/compare.py

```
@@ -0,0 +1,204 @@
+# ── shared helpers ─────────────────────────────────────────────────────────
+
+def all_cases():
+    return list(TEST_CASES) + [(u, None) for u in NO_MATCH_UTTERANCES]
+
+
+def compute_metrics(results, cases):
+    total     = len(cases)
+    match_n   = sum(1 for _, e in cases if e is not None)
+    nomatch_n = total - match_n
+    tp = fp = fn = tn = 0
+    per_tp = defaultdict(int); per_fn = defaultdict(int); per_fp = defaultdict(int)
```

Ruff (0.15.10)

[error] Multiple statements on one line (semicolon)

(E702)